projects/auth0-angular/src/lib/auth.config.ts

```diff
@@ -57,7 +60,19 @@ export interface HttpInterceptorRouteConfig {
    * '/api' - exactly match the route /api
    * '/api/*' - match any route that starts with /api/
    */
-  uri: string;
+  uri?: string;
```

Comment:
Now that this is optional, do we want anything in place to make sure either `uri` or `uriMatcher` is specified? What happens now if I don't give either setting?

Perhaps it just will never match anything, but maybe we should fail early in that case. Or at least present a warning.

Reply:
Ye that is a fair point. It couldn't stay required and moving uriMatcher to a different type didn't really solve anything. I will have a look how we can address this properly.

Reply:
I als had to change the way we check if it's an object or not (`isHttpInterceptorRouteConfig`): https://github.com/auth0/auth0-angular/pull/122/commits/4ae18cb55492bb7f2ec7fb4711554aa21d3bc694